alacritty.yml

```diff
@@ -205,6 +205,9 @@ key_bindings:
 mouse_bindings:
   - { mouse: Middle, action: PasteSelection }
 
+multiclick_threshold_ms: 300
```

Comment:
What do you think about putting these new properties into a struct? Maybe something like

```rust
selection:
  multiclick_threshold_ms: 300
  escape_characters: ",│`|:\"' ()[]{}<>"
```